src/ssh_session.ts

```diff
@@ -77,10 +77,11 @@ export class SSHSession extends BaseSession {
            .then(args => {
                 this.onSSHSession(args.session)
            }).catch(e => {
-                console.log("SSH pass startSession failed", e.toString())
-                if (e.toString().startsWith("Error: Not imp"))
+                const msg = e.toString()
+                console.log("SSH pass startSession failed", msg)
+                if (msg.startsWith("Error: Not imp"))
                     this.onStateChange("failed", Failure.NotImplemented)
-                else if (e.toString().startsWith("Error: Wrong password"))
+                else if (msg.match(/(Error: Wrong password|Auth fail)/))
```

Comment:
this is part of #393 , can we close that PR?

Reply:
It's a different check, and we manually copied that PR anyway